speech/unit_tests/test_client.py

```diff
@@ -109,12 +111,16 @@ def test_sync_recognize_content_with_optional_parameters(self):
         self.assertEqual(req['method'], 'POST')
         self.assertEqual(req['path'], 'speech:syncrecognize')
 
-        expected = SYNC_RECOGNIZE_RESPONSE['results'][0]['alternatives']
-        self.assertEqual(response, expected)
+        alternative = SYNC_RECOGNIZE_RESPONSE['results'][0]['alternatives'][0]
+        expected = [Transcript.from_api_repr(alternative)]
+        self.assertIsInstance(response[0], Transcript)
+        self.assertEqual(response[0].transcript, expected[0].transcript)
+        self.assertEqual(response[0].confidence, expected[0].confidence)
```

Comment:
What is the point of making `expected` if you just need to refer to `expected[0]`?